services/static-webserver/client/source/class/osparc/component/permissions/Permissions.js

```diff
@@ -38,6 +38,41 @@ qx.Class.define("osparc.component.permissions.Permissions", {
     this.__getCollaborators();
   },
 
+  statics: {
+    sortByAccessRights: function(a, b) {
+      const aAccessRights = a["accessRights"];
+      const bAccessRights = b["accessRights"];
+      if (aAccessRights["delete"] !== bAccessRights["delete"]) {
+        return bAccessRights["delete"] - aAccessRights["delete"];
+      }
+      if (aAccessRights["write"] !== bAccessRights["write"]) {
+        return bAccessRights["write"] - aAccessRights["write"];
+      }
+      if (aAccessRights["read"] !== bAccessRights["read"]) {
+        return bAccessRights["read"] - aAccessRights["read"];
+      }
+      return 0;
+    },
+
+    sortStudyOrServiceCollabs: function(a, b) {
+      const aAccessRights = a["accessRights"];
+      const bAccessRights = b["accessRights"];
+      let sorted = null;
+      if ("delete" in aAccessRights) {
+        // studies
+        sorted = this.self().sortByAccessRights(a, b);
```

Comment:
Q: so ``sorted`` is an integer  that resolves whether ``a<=b`` (==1:True, <=0:False)??

Reply:
-1, 1: sorting
0: no sorting